components/pedigree/resources/src/main/resources/pedigree/view/nodetypeSelectionBubble.js

```diff
@@ -460,7 +454,17 @@ define([], function(){
             this.expandedOptionsContainer.insert(svgContainer);
             this.expandedOptionsContainer.insert(plusBtn);
             this.expandedOptionsContainer.insert(createBtn);
+        },
+
+        generateSelectedNumberIcon: function(text) {
+            var textX = 35;
+            if (text.length > 1) {
+                textX = 25;
+            }
```

Comment:
@allasm Could [text-anchor](https://developer.mozilla.org/en/docs/Web/SVG/Attribute/text-anchor) be used instead of magic numbers?